pt-BR/faq.html

```diff
@@ -0,0 +1,181 @@
+---
+layout: page
+title: FAQ &lsaquo; Bisq - The decentralized Bitcoin exchange
+ref: faq
+lang: pt-BR
+flag: 🇧🇷
+language: Português (BR)
+outdated_translation: true
```

Comment:
```suggestion
outdated_translation: false
```

Pages marked as outdated show this message at the top of the page.

![Screenshot from 2019-11-10 18-12-02](https://user-images.githubusercontent.com/735155/68552490-a0e55680-03e5-11ea-8c7c-54bbb10bc919.png)